src/fee-accounts/components/notifications/fee-account-error-notification.component.html

```diff
@@ -0,0 +1,16 @@
+<ng-container *ngIf="errorMessages.length > 0">
```

Comment:
this should be a common component for error notification rather than just for a fee and pay. I think we already have it with the codebase.